templates/ldap.conf.j2

```diff
@@ -0,0 +1,60 @@
+<LDAP>
+	# LDAP server URL
+	URL		{{ ldap.url }}
+
+	{% if ldap.anonymous_bind == false %}
+	# Bind DN (If your LDAP server doesn't support anonymous binds)
+	BindDN		{{ ldap.bind_dn }}
+	# Bind Password
+	Password	{{ ldap.bind_password }}
+    {% endif %}
```

Comment:
It's a tab/spaces issue. The one line has 4 space, Github by default uses 8 spaces for each tab. I'm guessing your IDE only uses 4 spaces for each tab.